parser/model/ddl.go

```diff
@@ -222,13 +222,28 @@ type DDLReorgMeta struct {
 	SQLMode       mysql.SQLMode                    `json:"sql_mode"`
 	Warnings      map[errors.ErrorID]*terror.Error `json:"warnings"`
 	WarningsCount map[errors.ErrorID]int64         `json:"warnings_count"`
-	Location      *TimeZone                        `json:"time_zone"`
```

Comment:
We should keep the old field,  maybe mark it as deprecated instead of remove it.

Reply:
The main reason is that this field has not been used before, so I think it can be deleted directly